learnrl/memory.py

```diff
@@ -1,15 +1,15 @@
 import numpy as np
 from copy import copy
 import collections.abc as collections
-
+from learnrl import Agent
 
 class Memory():
 
     """
     A general memory for reinforcement learning agents
 
     Using the methods :meth:`remember` and :meth:`forget`
-    any :Class:`Agent` have a standardized :class:`Memory` !
+    any :class:`Agent` have a standardized :class:`Memory` !
```

Comment:
Here I think you must use `~learnrl.agent.Agent` and `~learnrl.memory.Memory` to obtain the desired behavior !